components/espressif/esp32-wroom-32.stanza

```diff
@@ -124,11 +124,12 @@ public pcb-module module (pn:esp32-wroom-32mpn, add-serial-programming-circuit?:
   port power : power
   pin boot
   pin en
+  pin gnd
```

Comment:
Why add a pin for gnd when that it should be incorporated in the power port?

Reply:
Just a convenience - many things attach to ground and I get tired of typing power-3v3.gnd